package BagLogic;

message PurchaseBagReq
{

}

message PurchaseBagResp
{
	required uint32 bag 				= 1;   //背包数
	required uint32 coin				= 2;	//银两
}
